common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -192,6 +193,16 @@ def _edit_problem_content(self):
             type_in_codemirror(self, 0, modified_content)
             modal.q(css='.action-save').click()
 
+    def _delete_student_state_for_problem(self):
+        with self._logged_in_session(staff=True):
+            self.courseware_page.visit()
+            staff_page = StaffPage(self.browser, self.course_id)
+            self.assertEqual(staff_page.staff_view_mode, 'Staff')
+            staff_page.q(css='a.instructor-info-action').nth(1).click()
+            staff_debug_page = StaffDebugPage(self.browser)
+            staff_debug_page.wait_for_page()
+            staff_debug_page.delete_state(self.USERNAME)
```

Comment:
Funny enough I have literally the exact same code already locally. Unfortunately it doesn't resolve the issue.